crates/ggml/src/format/loader.rs

```diff
@@ -146,8 +146,8 @@ pub fn load<E: Error, R: BufRead + Seek>(
 
     match container_type {
         ContainerType::Ggml
-        | ContainerType::Ggmf(1)
-        | ContainerType::Ggjt(1 | 2)
+        | ContainerType::Ggmf(1 | 100)
+        | ContainerType::Ggjt(1 | 2 | 100)
```

Comment:
Maybe use an enum here that maps these values to something more meaningful.

Reply:
That's a change I made, but I realise that with the growing number of versions this is only more confusing. I'll reintroduce constants soon